third_party/amd/lib/TritonAMDGPUToLLVM/ElementwiseOpToLLVM.cpp

```diff
@@ -261,9 +264,9 @@ Fp16_to_Fp8E5M2FNUZ_HW(Location loc, ConversionPatternRewriter &rewriter,
   return convert_val_Fp16_to_Fp8(loc, rewriter, v[0], v[1], "bf8");
 }
 
-ConverterT Fp16_to_Fp8E5M2FNUZ(int computeCapability) {
-  return computeCapability >= 300 ? Fp16_to_Fp8E5M2FNUZ_HW
-                                  : Fp16_to_Fp8E5M2FNUZ_SW;
+ConverterT Fp16_to_Fp8E5M2FNUZ(AMD::ISAFamily isaFamily) {
+  return isaFamily == AMD::ISAFamily::CDNA3 ? Fp16_to_Fp8E5M2FNUZ_HW
```

Comment:
I'm not trying to say that I have a better way to implement this. Just want to bring this up and see what you think.

Instead of querying ISAFamily, we can query "feature" here. What we want to know is whether the arch has hw instructions to do conversions. For now, it's only CDNA3. But in the future, we can have other archs. So maybe we can do something like 
getFeatures(isaFamily).contains(AMD::Feature::HW_CVT) ? Fp16_to_Fp8E5M2FNUZ_HW : Fp16_to_Fp8E5M2FNUZ_SW
Therefore, we don't have to change the logic here when we have a new arch that also has this feature.

Reply:
+1. That should be the direction to go. That's why I set up the new `TargetUtils.{h|cpp}` file. There might also existing utilities in the LLVM codebase that we can use.